dev-libs/glib/glib2-2.54.3.recipe

```diff
@@ -65,48 +64,62 @@ PROVIDES_devel="
 	cmd:gobject_query$secondaryArchSuffix
 	cmd:gtester$secondaryArchSuffix
 	cmd:gtester_report$secondaryArchSuffix
-	devel:libgio_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	devel:libglib_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	devel:libgmodule_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	devel:libgobject_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	devel:libgthread_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
+	devel:libgio_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
+	devel:libglib_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
+	devel:libgmodule_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
+	devel:libgobject_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
+	devel:libgthread_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
 	"
-
 REQUIRES_devel="
 	glib2$secondaryArchSuffix == $portVersion base
-	devel:libffi$secondaryArchSuffix
-	devel:libiconv$secondaryArchSuffix
-	devel:libintl$secondaryArchSuffix
-	devel:libpcre$secondaryArchSuffix
-	devel:libz$secondaryArchSuffix
-	cmd:perl
 	haiku$secondaryArchSuffix
-	lib:libffi$secondaryArchSuffix
+	cmd:perl
```

Comment:
I think it would be better to move `cmd:perl` to `BUILD_PREREQUIRES`.
(But I'm not sure about that, as I might be missing something.)

Hmm, I also think you don't need to mention any `lib:`in `REQUIRES_devel`, since these are provided by devel packages which already depend on their respective base packages.
In other words, if you put e.g. `devel:libffi$secondaryArchSuffix` in a `REQUIRES_devel` then you don't need to also put `lib:libffi$secondaryArchSuffix` since the former automatically pulls the latter.

Reply:
Tested already before i sent my PR, not worked, hp complains about non-declaredd dependencies, so i had to add them back. (The earlier recipe sports them too)

Reply:
Hmm, you are right. `lib:libintl` needs to be added explicitly. That's because it is provided by the `gettext_libintl` package, while `devel:libintl` is provided by the `gettext` package.

But I think you might be able to get rid of all the other `lib:` lines in your `REQUIRES_devel` and `BUILD_REQUIRES`.